### ISSUE TITLE
Not working on mobile

### ISSUE THREAD

**Issue by rodrigoportela** [dsebastien/obsidian-dataview-serializer#11]
I was able to use the plugin in a dektop app but on my Android device, although being able to install it and seeing no warning about an incompatible device (as it's the case for some other plugins that don't work on mobile), whenever I click the Enable option I get an error saying it wasn't possible to load the plugin. Isn't this plugin supposed to work on the mobile app? Or may there be another issue?

**Comment by dsebastien** [comment]
Hello

I haven't tested it on mobile, but this plugin doesn't do anything special, so I think it should work on mobile. One pre-req is to have the Dataview plugin also installed and enabled. Is it the case on your mobile device?

**Comment by rodrigoportela** [comment]
Yes, Dataview is installed and enabled on the mobile device.

**Comment by dsebastien** [comment]
I'll try connecting my Android device on my PC and check if I can reproduce/understand the issue

**Comment by quantumgardener** [comment]
I'm on iOS and I think the reason it doesn't work on mobile is the inclusion of the path in the link instead of just the page name 

For example on mobile I see [[/Quartz/notes/game.md]] when it should just be [[game]] to work properly.

**Comment by quantumgardener** [comment]
The path issue I thought was there on mobile isn't. A quick tap exposes the path. A long tap is needed to open the note and that works fine with filename in the path.

**Comment by merlinuwe** [comment]
Does not work on Android 11.

On activate: "Failed to load plugin"

**Comment by brsma** [comment]
Fails to load on iOS, too :-(

**Comment by dsebastien** [comment]
Damn, I wonder what breaks it on mobile. Can anyone take a look at the console log when the error happens?

**Comment by jaanonim** [comment]
Here is error form my android device (when activating plugin):
```
app.js:1 Plugin failure: dataview-serializer ReferenceError: process is not defined
    at anonymous (plugin:dataview-serializer:21:2801)
    at e.<anonymous> (app.js:1:2386641)
    at app.js:1:237226
    at Object.next (app.js:1:237331)
    at a (app.js:1:236049)
```

**Comment by jaanonim** [comment]
It looks like `immer` libary is trying to access `process` to check `NODE_ENV` witch don't exist on mobile, but I'm not sure about that.

**Comment by dsebastien** [comment]
Good catch!
I suppose it might be related to this: https://github.com/immerjs/immer/issues/1141

I'll probably just remove immer from the project. It has little added value here anyway.

**Comment by dsebastien** [comment]
I've released v1.4.1, trying to fix this by setting the `platform` property to `browser`. I think I mistakenly set it to `node` initially, thinking that it was the right choice given that Obsidian runs in Electron. But looking at the official plugin template again, I noticed they don't set the platform property, which means that it uses the default (browser).

Can you check if 1.4.1 fixes the issue, and reopen this ticket if the problem still exists?

**Comment by merlinuwe** [comment]
Thanks to your investigation it is now possible to install it on Android 11. Thank you very much!
